catalog/ui/catalog-ui-search/src/main/webapp/js/model/Query.js

```diff
@@ -497,7 +511,7 @@ define([
                 return _.pick(data, 'src', 'start', 'count', 'timeout', 'cql', 'sort', 'id');
             },
 
-            startSearch: function (forDeleted) {
+            startSearch: function (options) {
```

Comment:
Add a line to the beginning of this function 
```
options = _.extend({
    includeDeleted: false,
    includeHistoric: false
}, options);
```
Then you won't have to worry about them not being passed in by function callers, and it gives future developers a hint as to what parameters can be passed in.

Also suggest renaming them to with limit in the name rather than include, since it's misleading.